src/common/io.py

```diff
@@ -1218,6 +1218,13 @@ def __init__(self, fname, delayed_trajectory_loading = True):
                 all at the same time. Only works for files or streams that
                 are based on a file and has attribute name.
                 Default: True (for files)
+                
+            allow_file_updates --
+                If this is turned on, file updates (in terms of more
+                data point being added to the result file) is allowed.
```

Comment:
I think it is missing "s" in "data point**s**"

Reply:
Fixed!